### ISSUE TITLE
Error when hovering over a specific quest item

### ISSUE THREAD

**Issue by vaindil** [AeroScripts/QuestieDev#1404]
## Bug description
I'm currently on [Forbidden Knowledge](https://classic.wowhead.com/quest=737/forbidden-knowledge), which requires turning in [Patterned Bronze Bracers](https://classic.wowhead.com/item=2868/patterned-bronze-bracers). I made the bracers, but any time I hover over them in my bag I get the error below. This happens with all addons except for Questie disabled, so it does appear to be a Questie bug.

```
Message: Interface\AddOns\Questie\Modules\QuestieTooltips.lua:122: attempt to concatenate field 'text' (a nil value)
Time: Tue Oct 22 21:45:52 2019
Count: 40
Stack: Interface\AddOns\Questie\Modules\QuestieTooltips.lua:122: attempt to concatenate field 'text' (a nil value)
[C]: ?
Interface\AddOns\Questie\Modules\QuestieTooltips.lua:122: in function `GetTooltip'
Interface\AddOns\Questie\Modules\QuestieTooltips.lua:237: in function <Interface\AddOns\Questie\Modules\QuestieTooltips.lua:225>
[C]: ?
[C]: ?
[C]: ?
[C]: ?
[C]: ?
[C]: in function `SetBagItem'
Interface\FrameXML\ContainerFrame.lua:1318: in function `UpdateTooltip'
Interface\FrameXML\GameTooltip.lua:496: in function <Interface\FrameXML\GameTooltip.lua:475>
[C]: ?
[C]: ?
[C]: ?

Locals: 
```

## Questie version
5.0.0 7 fb7c60e

**Comment by Logonz** [comment]
This is fixed and hopefully similar thing with missing items won't happen again! 
Thanks for reporting the issue, fixed in 5.0.1

**Comment by vaindil** [comment]
Awesome, thanks for the quick response!
